lib/write.js

```diff
@@ -146,14 +146,7 @@ exports.usingStreaming = (deviceFileDescriptor, options = {}) => {
           transferredBytes += chunk.length;
 
           if (chunk.length < options.chunkSize) {
-            const finalChunkSize = _.attempt(() => {
-              if (chunk.length > options.minimumChunkSize) {
-                return options.chunkSize;
-              }
-
-              return options.minimumChunkSize;
-            });
-
+            const finalChunkSize = Math.ceil(chunk.length / options.minimumChunkSize) * options.minimumChunkSize;
```

Comment:
What happens if the chunk length is some very badly aligned number, like 127? If so, `finalChunkSize` will also be 127, which will for sure throw EINVAL. If the chunk length is smaller than the minimum chunk size, then we should pad with zeroes in order to get to that minimum chunk size.